pom.xml

```diff
@@ -90,8 +96,29 @@
 		<dependency>
 			<groupId>org.jenkins-ci.modules</groupId>
 			<artifactId>sshd</artifactId>
-			<version>1.6</version>
+			<version>2.2</version>
```

Comment:
I doubt it's a good idea. You target Jenkins core uses https://github.com/jenkinsci/jenkins/blob/jenkins-2.7.3/war/pom.xml#L137 . Does it come from upper bound requirements?

Reply:
actually not needed, you are right.